tests/store/test_sqlalchemy_store.py

```diff
@@ -646,9 +646,10 @@ def test_error_logging_to_deleted_run(self):
     def _search(self, experiment_id, filter_string=None,
                 run_view_type=ViewType.ALL, max_results=SEARCH_MAX_RESULTS_DEFAULT):
         search_filter = SearchFilter(filter_string=filter_string)
+        exps = [experiment_id] if isinstance(experiment_id, int) else experiment_id
+        print(self.store.search_runs(exps, search_filter, run_view_type, max_results))
```

Comment:
micronit: remove the `print`? or maybe we can compute the result before printing. the real underlying issue is that `self.store.search_runs(exps, search_filter, run_view_type, max_results)` is actually somewhat slow against a SQLite-file-backed store, so tests that use this API now runs kinda slowly.

Reply:
Good catch on that `print` thing. I was debugging -- forgot to remove. We shud add `print` check in that linter!

Reply:
slow search is a legit issue. The sql push-down should help by effectively not pulling out all runs only valid ones.